layers/cmd/pipeline.go

```diff
@@ -104,30 +103,11 @@ func (fork ForkStep) Run(ctx *PipelineContext) error {
 		errs[k] = nil
```

Comment:
Since it is serialized, we don't need errs anymore. Return the first error from the pipeline.